drafts/Stormless.md

```diff
@@ -0,0 +1,198 @@
+
+`sls stackstorm info --pack slack` 
+
+Oh my! There're so many what are they? I guess I need a PR to print action description. Meantme, `| grep admin` will get us the one we need: `slack.users.admin.invite`. Let's inquire the action for it's parameters:
+
+```
+$ sls stackstorm info --action slack.users.admin.invite
+slack.users.admin.invite ...... Send an invitation to join a Slack Org
+Parameters
+  attempts [integer]  ......... description is missing
+  channels [string]  .......... Channels to auto join.
+  email [string] (required) ... Email address to send invitation to.
+  first_name [string]  ........ Users first name
+  set_active [boolean]  ....... Should the user be active.
+  token [string]  ............. Slack API token.
+Config
+  action_token [string]  ...... Slack Action token.
+  admin [object]  ............. Admin-action specific settings.
+  post_message_action [object]   Post message action specific settings.
+  sensor [object]  ............ Sensor specific settings.
+```
+
+
+Awesome! We can see that there is only one required parameter, `email`, but I'll add `first_name` to be conversational. The token can be passed as parameters, or as config. And if I choose to use config, my prior tribal knowledge hints that the `admin [object]` requires only `admin_token`. The very one I asked you to remember when you were setting up Slack workspace.
+> PRO TIP. While we are still polishing the plugin to expose all the Config details, you can find it out by exploring StackStorm Exchange pack config schema in `config.schema.yaml` file. For example, here is [config.example.yaml](https://github.com/StackStorm-Exchange/stackstorm-slack/blob/master/config.schema.yaml#L47-L78) for our Slack pack.
+
+Now we have all we need to create the heart of any Serverless project: the `serverless.yml`. Here it comes: 
+
+```
+service: signup-stormless
+
+provider:
+  name: aws
+  runtime: python2.7
+
+functions:
+  InviteSlack:
+    events:
+      - http:
+          method: POST
+          path: invite
+    stackstorm:
+      action: slack.users.admin.invite
+      input:
+        email:       "{{ input.body.email }}"
+        first_name:  "{{ input.body.first_name }}"
+      config:
+        admin: ${file(env.yml):slack}
+      output:
+        statusCode: 200
+        body: "{{ output }}"
+
+plugins:
+  - serverless-plugin-stackstorm
+```
+
+This is a good time to learn a bit of Serverless. The quick pointers below won't
+reading the docs to understand
+
+I threw in the `events` section here so that we can test the function with REST call to an AWS API Gateway endpoint.
+Note that this default configuration instructs API Gateway to pass the REST POST call with POST body under the `body` key ([details here](TODO-insert-link)). When we POST `{"first_name": "Santa", 
```

Comment:
Don't forget to insert this link

Reply:
Maybe not a bad idea to add a short explanation on what API gateway is and how it helps here. I have the context so I know what is for, but maybe others won't, dunno.

Reply:
If we're assuming that reader may not know sls\lambda basics, this article should be a fair bit longer.

Reply:
we assume they are curious and motivaeted to learn. 
Thus links as pointers to right places to accelerate, but not to teach everything.

Reply:
👍